### PR TITLE
WIP: first steps towards plotting the hospitalisierungsrate

### PR DESCRIPTION
**Pull request by fangohr**

### CODE REVIEW

**Comment by fangohr** [comment]
It is a start. We could put this online, and gradually improve on it.

**Review by kirienko — COMMENTED** [review]
One natural question: is it okay that you added only one `.nbdata` file while there are two `.ipynb` files?

Honestly, I didn't have time for the review today, sorry. I'll try to do it tomorrow (most likely in the evening).

**Comment by fangohr** [comment]
> One natural question: is it okay that you added only one `.nbdata` file while there are two `.ipynb` files?

Good question! 

I have created two notebooks: one to display, and one to help contain some lengthy code that would distract from the plots. The one to display (`germany-hospitalisierungsrate.ipynb`) will call the other one (`germany-compute-hospitalisierungsrate.ipynb`) in the first cell to define data and convenience functions.

The `germany-hospitalisierungsrate.nbdata` file tells pelican to use `germany-hospitalisierungsrate.ipynb` as a page to include in the html output. I thought it might be okay / good to ignore the other notebook file here (i.e. no nbdata file for that) as we do not want to show that so aggressively. (But that's open for debate. We could add a URL to the location of the compute notebook on github - so that really curious people can find it.) 

I hope that explains it.

File: tools/pelican/content/ipynb/germany-compute-hospitalisierungsrate.ipynb [hunk 1]
```diff
@@ -0,0 +1,4575 @@
+{
```
Comment (kirienko) [inline tools/pelican/content/ipynb/germany-compute-hospitalisierungsrate.ipynb:1]: <p>I would either uncomment or remove this part.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oscovida/oscovida/pull/282/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (kirienko) [inline tools/pelican/content/ipynb/germany-compute-hospitalisierungsrate.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #14.        # and oll other länder
</pre><p>typo? (<code>all</code>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oscovida/oscovida/pull/282/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (kirienko) [inline tools/pelican/content/ipynb/germany-compute-hospitalisierungsrate.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #51.    # fig, ax = plot_hi(data["2021-10-01":])
</pre><p>I would clean this commented line up</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oscovida/oscovida/pull/282/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

**Review by kirienko — APPROVED** [review]
In general that looks good! I especially love the solution with different color zones for different values.

I would also add some notes on what could be improved if you have more spare time :smiley: 

image

1. that `???` is not too self-explanatory. My first thought was that it's a kind of a render bug. Maybe we can add more text here? Dunno, "The government hasn't decided yet, but we expect no good" :smile: 
2. The text on the right looks a little messy. But, to be honest, as far as I know, there's no easy solution of this problem if we keep using `pyplot` (`plotly` provides cleaner solutions, like annotations when you hover over the graph, but I don't know if it is `pelican`-friendly).

Also I see some error message from `jupyter`, but it might be a local problem of my installation:

```bash
ERROR:asyncio:Exception in callback <TaskWakeupMethWrapper object at 0x7fca9d2fcf40>(<Future finis...C: 1\r\n\r\n'>)
handle: <Handle <TaskWakeupMethWrapper object at 0x7fca9d2fcf40>(<Future finis...C: 1\r\n\r\n'>)>
Traceback (most recent call last):
  File "/home/kir/miniconda3/envs/oscovida/lib/python3.8/asyncio/events.py", line 81, in _run
    self._context.run(self._callback, *self._args)
RuntimeError: Cannot enter into task <Task pending name='Task-4' coro=<HTTP1ServerConnection._server_request_loop() running at /home/kir/miniconda3/envs/oscovida/lib/python3.8/site-packages/tornado/http1connection.py:823> wait_for=<Future finished result=b'GET /api/co...PC: 1\r\n\r\n'> cb=[IOLoop.add_future.<locals>.<lambda>() at /home/kir/miniconda3/envs/oscovida/lib/python3.8/site-packages/tornado/ioloop.py:688]> while another task <Task pending name='Task-27' coro=<MappingKernelManager.start_kernel() running at /home/kir/miniconda3/envs/oscovida/lib/python3.8/site-packages/notebook/services/kernels/kernelmanager.py:176> cb=[IOLoop.add_future.<locals>.<lambda>() at /home/kir/miniconda3/envs/oscovida/lib/python3.8/site-packages/tornado/ioloop.py:688]> is being executed.
```

File: tools/pelican/content/ipynb/germany-compute-hospitalisierungsrate.ipynb (hunk 1 above)
Comment (fangohr) [reply tools/pelican/content/ipynb/germany-compute-hospitalisierungsrate.ipynb:1]: <p>I have left this in the notebook, but added an explanation at the top of the notebook, why this is in there.</p>

Comment (fangohr) [reply tools/pelican/content/ipynb/germany-compute-hospitalisierungsrate.ipynb:1]: <p>Thanks, fixed.</p>

Comment (fangohr) [reply tools/pelican/content/ipynb/germany-compute-hospitalisierungsrate.ipynb:1]: <p>Also left this in as a quick way of trying the plot when developing it further. (See new entry in first cell of notebook.)</p>

**Comment by fangohr** [comment]
I have replaced the '???' with a more sensible text.

**Pull request merged**